src/icons/icons.css

```diff
@@ -0,0 +1,19 @@
+#omrs-logo-full-color,
```

Comment:
This entire file can be deleted, at least for now.

This was how we were changing from colored logos to mono colored logo. But that doesn't apply to icons.